apps/Gruntfile.js

```diff
@@ -297,7 +297,7 @@ module.exports = function (grunt) {
       cmd += ' --cachefile ' + outputDir + options.cacheFile;
     }
     cmd += ' --extension=.jsx' +
-      ' -t [ babelify --compact=false --sourceMap --sourceMapRelative="$PWD" ]' +
+      ' -t [ babelify --compact=false --sourceMap ]' +
```

Comment:
Based on a comment in babelify, sourceMapRelative became the default in babel v6.0 and this option is now ignored.